controllers/helmrelease_controller.go

```diff
@@ -328,14 +328,14 @@ func (r *HelmReleaseReconciler) reconcileRelease(ctx context.Context,
 		// Fail if install retries are exhausted.
 		if hr.Spec.GetInstall().GetRemediation().RetriesExhausted(hr) {
 			err = fmt.Errorf("install retries exhausted")
-			return v2.HelmReleaseNotReady(hr, released.Reason, released.Message), err
+			return v2.HelmReleaseNotReady(hr, released.Reason, err.Error()), err
```

Comment:
This (and the change below) are required because otherwise the long error message will still end up in the `Ready` condition.

It will also make the error more visible to users, as I know from providing support to people that it is a bit hidden at the moment.